_search-plugins/retrieve-specific-fields.md

```diff
@@ -0,0 +1,866 @@
+
+---
+
+#### Table of contents
+1. TOC
+{:toc}
+
+
+---
+
+## Disabling `_source`
+
+You can include `"_source": false` line in the search request to prevent the `_source` field from being included in the response. See following example:
+
+```
+GET "/index1/_search?pretty"
+{
+    "_source": false,
+    "query": {
+        "match_all": {}
+  }
+}
+```
+
+As no fields were selected in the previous search, the retrieved hits will only include `_index`, `_id` and `_score` of the hits. As can be seen in the following example:
+
+```
+{
+  "hits" : {
+    "total" : {
+      "value" : 2,
+      "relation" : "eq"
+    },
+    "max_score" : 1.0,
+    "hits" : [
+      {
+        "_index" : "index1",
+        "_id" : "41",
+        "_score" : 1.0
+      },
+      {
+        "_index" : "index1",
+        "_id" : "51",
+        "_score" : 1.0
+      }
+    ]
+  }
+}
+```
+
+## Specifying the fields to retrieve
+
+You can list the fields of interest in the search request using `fields` parameter. Wildcard patterns are also accepted. See following example:
+
+```
+GET "/index1/_search?pretty"
+{
+    "_source": false,
+    "fields": ["age", "nam*"],
+    "query": {
+        "match_all": {}
+  }
+}
+```
+
+Example response:
+
+```
+{
+  "hits" : {
+    "total" : {
+      "value" : 2,
+      "relation" : "eq"
+    },
+    "max_score" : 1.0,
+    "hits" : [
+      {
+        "_index" : "index1",
+        "_id" : "41",
+        "_score" : 1.0,
+        "fields" : {
+          "name" : [
+            "John Doe"
+          ],
+          "age" : [
+            30
+          ]
+        }
+      },
+      {
+        "_index" : "index1",
+        "_id" : "51",
+        "_score" : 1.0,
+        "fields" : {
+          "name" : [
+            "Jane Smith"
+          ],
+          "age" : [
+            25
+          ]
+        }
+      }
+    ]
+  }
+}
+```
+
+### Extracting fields with custom format
+
+You can also use object notation, to apply a custom format to the chosen field, see following example.
+
+If you have the following document:
+
+```
+{
+  "_index": "my_index",
+  "_type": "_doc",
+  "_id": "1",
+  "_source": {
+    "title": "Document 1",
+    "date": "2023-07-04T12:34:56Z"
+  }
+}
+```
+
+You can query with `fields` parameter and custom format:
+
+```
+GET /my_index/_search
+{
+  "query": {
+    "match_all": {}
+  },
+  "fields": {
+    "date": {
+      "format": "yyyy-MM-dd"
+    }
+  },
+  "_source": false
+}
+```
+
+Additionally, you can also use [most fields]({{site.url}}{{site.baseurl}}/query-dsl/full-text/multi-match/#most-fields) and [field aliases]({{site.url}}{{site.baseurl}}/field-types/supported-field-types/alias/) with `fields` parameter, as it queries both document `_source` and `_mappings` of the index.
+
+## Searching with `docvalue_fields`
```

Comment:
Let's also include a line that `_source` field is not always enabled as it can be disabled at index time using index mapping - 
```json
  "mappings": {
    "_source": {
      "enabled": false
    }
  }
```
In such scenario these alternate approaches  (`docvalue_fields`, `stored fields`) come in handy.

Reply:
Thats added now